pkg/urbit/noun/manage.c

```diff
@@ -94,19 +94,29 @@ _cm_punt(u3_noun tax)
 }
 #endif
 
+static void _write(int  fd,  const  void  *buf,  size_t count)
```

Comment:
Please use c3_assert(), u3l_log(), and adjust the whitespace to follow surrounding conventions.

Reply:
done